src/remotion/FluidText/FluidTextTemplate.tsx

```
@@ -25,13 +28,71 @@ export const FluidTextTemplate: React.FC<FluidTextProps> = ({
     const totalDuration = duration * fps;
 
     const characters = text.split('');
 
+    // Create different fluid motion patterns
+    const getFluidTransform = (index: number) => {
+        const baseTime = frame / (waveSpeed * 3);
+        const charOffset = index / 2;
+        
+        switch (fluidType) {
+            case 'wave':
+                const wave = Math.sin(baseTime + charOffset) * waveIntensity;
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)